app/src/frontend/data_fields.ts

```diff
@@ -1,18 +1,71 @@
 export enum Category {
     Location = 'Location',
-    LandUse = 'Land Use',
+    LandUse = 'LandUse',
     Type = 'Type',
     Age = 'Age',
-    SizeShape = 'Size & Shape',
+    SizeShape = 'SizeShape',
     Construction = 'Construction',
     Streetscape = 'Streetscape',
     Team = 'Team',
     Sustainability = 'Sustainability',
     Community = 'Community',
     Planning = 'Planning',
-    Like = 'Like Me!'
+    Like = 'Like',
+
+    Unknown = 'Unknown'
 }
```

Comment:
The string enum values now reflect the name of the key. The human-readable values were moved to the separate `categories` definition object which also contains the slug values